nengo_loihi/builder/inputs.py

```diff
@@ -43,56 +41,53 @@ def receive(self, t, x):
 
 
 class ChipReceiveNode(Node):
-    """For receiving host->chip messages"""
+    """Represents chip end-point for host->chip encoded connections.
+
+    This Node does not do anything, other than act as a placeholder in ``Connection``s.
+
+    Attributes
+    ----------
+    spike_target : SpikeInput
+        Spike input that will inject the spikes from the sender onto the chip.
+    """
 
     def __init__(self, dimensions, size_out, label=Default):
         self.raw_dimensions = dimensions
-        self.spikes = []
-        self.spike_input = None  # set by builder
+        self.spike_target = None
         super(ChipReceiveNode, self).__init__(
             self.update, size_in=0, size_out=size_out, label=label
         )
 
-    def clear(self):
-        self.spikes.clear()
-
-    def receive(self, t, x):
-        assert len(self.spikes) == 0 or t > self.spikes[-1][0]
-        assert x.ndim == 1
-        self.spikes.append((t, x.nonzero()[0]))
-
     def update(self, t):
         raise SimulationError("ChipReceiveNodes should not be run")
```

Comment:
Minor nitpick: `ChipReceiveNodes` -> `type(self).__name__` since this method is being inherited.

Reply:
Good call! Will change.

Reply:
Done in 42cd76eb